Landing page visual and under the hood improvements

samualon opened the pull request:
Hi!

I stumbled upon your project, AutoSpoto, and was intrigued by the app idea.

While I was searching for some VueJS project to contribute to, I found your landing page. 

I decided to build on the existing design and *finish it up* a bit. Additionally, I fixed some errors and warning that the code produced and optimized some minor bugs. A more comprehensive list of changes can obviously be found in the commits.

I did this solely out of practice, it's up to you if you decide to incorporate the changes. If you're not completely satisfied with the result, don't hesitate to comment on this PR and I'll try to improve it.

samualon committed:
:art: Change div with function calls to RouterLinks

samualon committed:
:art: Improve footer style coding

samualon committed:
:art: Change logo link to RouterLink and remove dead code

samualon committed:
:lipstick: Tweak footer spacing and alignement

samualon committed:
:art: Make header into separate component

samualon committed:
:art: Make footer into separate component

samualon committed:
:bug: Fix double createApp in main.js

samualon committed:
:pencil2: Move GH link methods to correct component

samualon committed:
:pencil2: Fix class instead of div typo

samualon committed:
:pencil2: Fix tab

samualon committed:
:art: Remove newlines

samualon committed:
:lipstick: Tweak background color to match video

samualon committed:
:lipstick: Remove horizontal lines

samualon committed:
:lipstick: Add new middle section design

samualon committed:
:lipstick: Minor styling adjustments

samualon committed:
:lipstick: Tweak footer styling

samualon committed:
:lipstick: Tweak header styling

samualon committed:
:lipstick: Tweak main card styling

samualon committed:
:lipstick: Remove navbar border

samualon committed:
:lipstick: Make navbar fixed

samualon committed:
:art: Tweak navbar logo spacing

martinmaly21 commented:
This is awesome! This layout looks way better.

We really appreciate you taking the time to do this :)


Just one thing that need to be fixed up before we merge this in:
1. The logo appears to be stretched in the nav bar on my screen:
<img width="2056" alt="Screenshot 2024-09-06 at 5 59 33 PM" src="https://github.com/user-attachments/assets/fab83567-4056-49cc-92b6-03422e7500e4">


Bonus: This container looks great, but it would be cool if we could give it some rounded corners :)

<img width="1765" alt="Screenshot 2024-09-06 at 6 00 09 PM" src="https://github.com/user-attachments/assets/97e607a7-3455-47b4-816a-ef06c13fa0d4">

samualon committed:
:bug: Fix logo ratio in nav

samualon committed:
:lipstick: Add rounded corners to story card

samualon commented:
It's my pleasure to help out!

The logo was fixed and I added round corners to the story card :)

One thing I noticed from your screenshots is that the video frame again seems to have a slighty different shade than the background. I noticed this on the original version and assumed that the video had a slighty different shade, so I adjusted the background of the site to match it. On my machine it seems to match perfectly, but on your screenshot it seems to differ. So I'm not completely sure how to fix that.

![image](https://github.com/user-attachments/assets/35f86b93-698e-479c-ab34-74346284d3f1)

martinmaly21 commented:
Awesome! Everything looks great. The video frame being a slightly different colour is OK!

Merging now :)

Thanks again for your help!
